src/Fantomas.Tests/ComputationExpressionTests.fs

```diff
@@ -1609,25 +1609,25 @@ let rec loop () =
 
     match msg with
     | Handle (eventSource, command, reply) ->
-        let! stream = eventSource |> eventStore.GetStream
```

Comment:
Looks sane to me

Reply:
This is actually a bit of a side effect of a change in CodePrinter.
I saw the current behavior is somewhat weird and logged https://github.com/fsprojects/fantomas/issues/1501 as it is not in line with the style guides.
So, the modified tests are more a result of that change.